build/testStartupScriptGenerators.sh

```diff
@@ -31,4 +31,5 @@ docker run -v $GEN_DIR:$GEN_DIR_CONTAINER_RO:ro --name $CONTAINER_NAME golang:1.
 	./restorePackages.sh && \
 	echo && \
 	echo Running tests... && \
-	go test ./... -v"
\ No newline at end of file
+	chmod u+x testPackages.sh && \
```

Comment:
We should be good to pull the latest from `main` to get Paul's merged changes for Go